src/inverseRobot.js

```diff
@@ -14,17 +14,30 @@
  * Example:
  *
  * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+ * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+ *
+ * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
  * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
- * inverseRobot(robert) === null
- * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
  *
+ * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
+ * inverseRobot(robert) === null
  *
  * @param {object} robot
  *
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    result[robot[key]] = key;
+  }
+
+  if (Object.keys(robot).length !== Object.keys(result).length) {
+    return null;
+  }
```

Comment:
```suggestion
  for (const key in robot) {
    result[robot[key]] = key;
  }

  if (Object.keys(robot).length !== Object.keys(result).length) {
    return null;
  }
```
use hasOwnProperty
```
    ```
 const value = robot[key];

     if (newRobot.hasOwnProperty(value)) {
       return null;
     }
```
```

Reply:
I haven't understood anything

Reply:
I've understood nothing